classes/book.rb

```diff
@@ -0,0 +1,16 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
```

Comment:
- [ ] I think it would be better if you update the `#add_rental` method as it should include `person`, `date` as parameters and the `book` itself should be sent as a parameter to create the new rental achieving the **'has-many' association**.  As for adding new rental you will need all listed parameters(`person`. `data`, and `book`). You can edit to something like this:
```ruby
   ...
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```

Reply:
@tresorsawasawa It's implemented with another approach, I think this one might be more understandable, but the requirement is fulfilled. I will use this one in a different repo though.